src/caf/toolkit/io.py

```diff
@@ -229,3 +230,73 @@ def read_csv_matrix(
             matrix = matrix.reindex(matrix.columns, axis=0)
 
     return matrix
+
+
+def find_file_with_name(
+    folder: pathlib.Path, name: str, suffixes: collections.abc.Sequence[str]
+) -> pathlib.Path:
+    """Find file in `folder` based on acceptable `suffixes`.
```

Comment:
Pretty sure best practice is to not include `argument` names in the short description.

```suggestion
    """Find a file in a folder matching _any_ acceptable suffix.
```